creator/extract_configs/templates/sample_manifest_config.py

```diff
@@ -0,0 +1,126 @@
+"""
+This is an extract configuration for a Sample Manifest
+
+See: https://www.notion.so/d3b/Internal-File-Types-and-Guidelines-6edd92087f04445e84a236202831edbd#aa2c7d8a50164a08ab685b42b784293e
+
+Required Columns:
+    Participant ID
+    Sample ID
+    Sequencing Center
+
+Optional Columns:
+    Aliquot ID
+    Tissue Type
+    Composition
+    Body Site Name
+    Body Site Code
+    Age at Collection
+    Age at Collection Units
+    Descriptor
+    Analyte Type
+    Method of Sample Procurement
+    Volume
+    Volume Units
+    Concentration
+    Concentration Units
```

Comment:
Analyte is required not optional.

And what is the plan for when the research group sends a piece of tissue to a sequencing center and then the sequencing center sends part of that tissue to another sequencing center to sequence the same or different analyte or both without assigning new identifiers? This has happened before and will happen again.